core/storage/audit/gae_models_test.py

```diff
@@ -16,16 +16,20 @@
 
 """Test for audit models."""
 
-from core.platform import models
```

Comment:
Hm. I know I told you something different earlier, but looking at this and thinking about it more, how do you feel about having all model imports always go through models.Registry()? It is a simpler rule for devs to follow than "do it for only non-storage models" and I think it's easier to enforce, so maybe we should just do that throughout the codebase instead. It also keeps things uniform throughout the codebase. Would that be OK?

Reply:
I mean I'm okay with loading it through the registry everywhere, but I think there are some disadvantages like that the pylint will silently skip some things because it doesn't recognize the loading through the registry and also that it is more prone to error, because Python compiler will fail if we use classic import and do something like `base_model.non_existent_method()` but will not fail if we use the registry approach.

Reply:
Good points, thanks!

I think these issues are alleviated with the 100% backend coverage requirement, though -- so I'm fine with the tradeoff in this case, in favour of having consistency throughout the codebase. So yeah, let's use the registry everywhere.

Reply:
Done.